microservices-orders/docs/index.rst

```diff
@@ -121,137 +130,132 @@ For more learning on Kafka Streams API that you can use as a reference while wor
+   .. codewithvars:: bash
 
-* Exercise 1: Persist events 
-* Exercise 2: Event-driven applications
-* Exercise 3: Enriching streams with joins
-* Exercise 4: Filtering and branching
-* Exercise 5: Stateful operations
-* Exercise 6: State stores
-* Exercise 7: Enrichment with |ksqldb| 
+      cd examples/microservices-orders
+      git checkout |release_post_branch|
 
-For each exercise:
+#. Run the full end-to-end working solution to see a customer-representative deployment of a streaming application. This requires no code development; it just provides context for each of the exercises in which you will develop pieces of the microservices.
 
-#. Read the description to understand the focus area for the exercise
-#. Edit the file specified in each exercise and fill in the missing code
-#. Copy the file to the project, then compile the project and run the test for the service to ensure it works
+   - Exercise 0: Run end-to-end demo
 
+#. After you have successfully run the full solution, go through each of the execises 1-7 to better understand the basic principles of streaming applications:
+
+   - Exercise 1: Persist events 
+   - Exercise 2: Event-driven applications
+   - Exercise 3: Enriching streams with joins
+   - Exercise 4: Filtering and branching
+   - Exercise 5: Stateful operations
+   - Exercise 6: State stores
+   - Exercise 7: Enrichment with |ksqldb| 
+
+#. For each of the above exercises:
+
+   - Read the description to understand the focus area for the exercise
+   - Edit the file specified in each exercise and fill in the missing code
+   - Copy the file to the project, then compile the project and run the test for the service to ensure it works
 
 Exercise 0: Run end-to-end demo
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Running the fully working demo end-to-end provides context for each of the later exercises.
 
-To run this demo end-to-end, you have one of three modes to choose from: 
-
-* |cp| locally
-* Docker 
-* |ccloud|
+#. Ensure you've followed the appropriate prerequisites section above prior to starting.
 
-Ensure you've followed the appropriate prerequisites section above prior to starting.
+#. Start the demo in one of three modes, depending on whether you are running with |ccloud|, in Docker, or |cp| locally:
 
-#. Start the demo
-
-   * If you are running |cp| locally, then run the full solution (this also starts a local |cp| cluster using Confluent CLI):
+   * |ccloud|: first log in to |ccloud| with the command ``ccloud login``, and use your |ccloud| username and password. To prevent being logged out, use the ``--save`` argument which saves your |ccloud| user login credentials or refresh token (in the case of SSO) to the local ``netrc`` file. Then run the full solution (this starts a new |ccloud| environment and Kafka using the :devx-examples:`ccloud stack library|ccloud/ccloud-stack/README.md` of this repository).
```

Comment:
```suggestion
   * |ccloud|: first log in to |ccloud| with the command ``ccloud login``, and use your |ccloud| username and password. To prevent being logged out, use the ``--save`` argument which saves your |ccloud| user login credentials or refresh token (in the case of SSO) to the local ``netrc`` file. Then run the full solution using the provided script (this starts a new |ccloud| environment and Kafka using the :devx-examples:`ccloud stack library|ccloud/ccloud-stack/README.md` of this repository).
```